client/src/components/MeetingExperience.tsx

```diff
@@ -46,11 +54,12 @@ export const MeetingExperience = (props: MeetingExperienceProps): JSX.Element =>
     userId,
     waitingSubtitle,
     waitingTitle,
+    postCall,
     onDisplayError
   } = props;
 
   const [callWithChatAdapter, setCallWithChatAdapter] = useState<CallWithChatAdapter | undefined>(undefined);
-
+  const [renderPostCall, setRenderPostCall] = useState(false);
```

Comment:
Can we put type in for `useState`?

`const [renderPostCall, setRenderPostCall] = useState<boolean>(false);`

Reply:
Thanks! Have made this change..